src/utils/parsers/common.ts

```diff
@@ -148,15 +148,18 @@ export abstract class BaseParser {
         
         for (const line of file.lines) {
             index += line.content.length - 1
+            if (line.content.replace(/ /g, '').startsWith(this.commentCharacters[0])) {
```

Comment:
You could use `.trim()` to handle any kind of whitespace character. Also, shouldn't this check for any of the commentCharacters, not just the first one?